pkg/config/schema_test.go

```diff
@@ -0,0 +1,63 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+var (
+	schema            = gojsonschema.NewStringLoader(requestCapabilities)
+	testFailingInputs = []string{
+		`{}`,                                                                                                                                                                   // App missing
+		`{"app": {}}`,                                                                                                                                                          // Empty App object
+		`{"app": {"version": "6.4.1"}}`,                                                                                                                                        // Device missing
+		`{"app": {"version": "6.4.1"}, "device": {}, "os": {"platform": "WatchOS", "version": "9.4"}}`,                                                                         // Empty Device object
+		`{"app": {"version": "6.4.1"}, "device": {"location": {}, "os": {"platform": "WatchOS", "version": "9.4"}}}`,                                                           // Empty Location object
+		`{"app": {"version": "6.4.1"}, "device": {"location": {"locale": "en_GB"}, "os": {"platform": "WatchOS", "version": "9.4"}}}`,                                          // TimezoneOffset missing
+		`{"app": {"version": "6.4.1"}, "device": {"location": {"locale": "en_GB", "timezoneOffset": 7200}}}`,                                                                   // OS missing
+		`{"app": {"version": "6.4.1"}, "device": {"location": {"locale": "en_GB", "timezoneOffset": 7200}, "os": {}}}`,                                                         // Empty Os object
+		`{"app": {"version": "6.4.1"}, "device": {"location": {"locale": "en_GB", "timezoneOffset": 7200}, "os": {"platform": "WatchOS"}}}`,                                    // Version missing
+		`{"app": {"version": "6.4.1"}, "device": {"location": {"locale": "testLocale", "timezoneOffset": 7201}, "os": {"platform": "WatchOS", "version": "9.4"}}}`,             // Wrong locale format
+		`{"app": {"version": "6.4.1"}, "device": {"location": {"locale": "testLocale", "timezoneOffset": 7201}, "os": {"platform": "WatchOS", "version": "9.4"}}, "user": {}}`, // Empty User object
+	}
+	testSuccessInputs = []string{
+		`{"app": {"version": "6.4.1"}, "device": {"location": {"locale": "en_GB", "timezoneOffset": 7201}, "os": {"platform": "WatchOS", "version": "9.4"}}, "user": {"age": 23}}`, // Working case
+	}
+)
+
+func TestPayloadValidationFailure(t *testing.T) {
+	var (
+		want = "invalid JSON error"
+	)
```

Comment:
For a single variable it's almost always preferrable to use the short assignment (`:=`) as this takes up too much space for no gain.

Reply:
of course.  the compile error makes sense now. I was missing the `:=`